.azure-pipelines/pull_request.yml

```diff
@@ -14,7 +14,7 @@ resources:
     name: scipp/pipelines
     type: github
     endpoint: scipp
-    ref: main
+    ref: gcc11  # TODO revert to main
```

Comment:
Why do we need this change?

Reply:
Why hasn't https://github.com/scipp/scipp/blob/gcc11/conda/meta.yaml#L12 changed?

Reply:
Now I am confused. Why do we have two `meta.yaml`, one in scipp and one in pipelines?

Reply:
The one on `pipelines` is used for testing in the `pipelines` repo, iirc.

Reply:
... and should have no effect on the scipp builds.